ui/src/components/credentials/LinkDetails.tsx

```diff
@@ -0,0 +1,250 @@
+import { Button, Card, Space, Typography } from "antd";
+import { useCallback, useEffect, useState } from "react";
+import { generatePath, useNavigate, useParams } from "react-router-dom";
+import { z } from "zod";
+
+import { APIError } from "src/adapters/api";
+import { getLink } from "src/adapters/api/credentials";
+import { getJsonSchemaFromUrl } from "src/adapters/jsonSchemas";
+import { getAttributeValueParser } from "src/adapters/parsers/jsonSchemas";
+import { ReactComponent as IconTrash } from "src/assets/icons/trash-01.svg";
+import { LinkDeleteModal } from "src/components/credentials/LinkDeleteModal";
+import { ObjectAttributeValueTree } from "src/components/credentials/ObjectAttributeValueTree";
+import { Detail } from "src/components/shared/Detail";
+import { ErrorResult } from "src/components/shared/ErrorResult";
+import { LoadingResult } from "src/components/shared/LoadingResult";
+import { SiderLayoutContent } from "src/components/shared/SiderLayoutContent";
+import { useEnvContext } from "src/contexts/Env";
+import { Link, ObjectAttribute, ObjectAttributeValue } from "src/domain";
+import { ROUTES } from "src/routes";
+import {
+  AsyncTask,
+  hasAsyncTaskFailed,
+  isAsyncTaskDataAvailable,
+  isAsyncTaskStarting,
+} from "src/utils/async";
+import { isAbortedError, makeRequestAbortable } from "src/utils/browser";
+import { CREDENTIALS_TABS } from "src/utils/constants";
+import { processZodError } from "src/utils/error";
+import { formatDate } from "src/utils/forms";
+
+export function LinkDetails() {
+  const navigate = useNavigate();
+  const { linkID } = useParams();
+
+  const env = useEnvContext();
+
+  const [credentialSubjectValue, setCredentialSubjectValue] = useState<
+    AsyncTask<ObjectAttributeValue, string | z.ZodError>
+  >({
+    status: "pending",
+  });
+  const [link, setLink] = useState<AsyncTask<Link, APIError>>({
+    status: "pending",
+  });
+
+  const [showModal, setShowModal] = useState<boolean>(false);
+
+  const fetchJsonSchemaFromUrl = useCallback(({ link }: { link: Link }): void => {
+    setCredentialSubjectValue({ status: "loading" });
+
+    void getJsonSchemaFromUrl({ url: link.schemaUrl }).then(([jsonSchema]) => {
+      const credentialSubject =
+        (jsonSchema.type === "object" &&
+          jsonSchema.schema.properties
+            ?.filter((child): child is ObjectAttribute => child.type === "object")
+            .find((child) => child.name === "credentialSubject")) ||
+        null;
+
+      if (credentialSubject) {
+        const parsedCredentialSubject = getAttributeValueParser(credentialSubject).safeParse({
+          ...link.credentialSubject,
+          // Add required id property that links lack
```

Comment:
Is this a BE issue? Either way we need to create one and reference it here.

Reply:
This has changed in the last review of the code [per a suggestion](#discussion_r1176430874) from @adrifdez. Now instead of artificially adding a value, we're artificially removing the `id` attribute from the JSON Schema, which apart from being more correct, has the beneficial side effect of removing the attribute from the tree.

We do not have to create any issues for this because there is nothing to solve or fix. This is expected to be like this. The reason is that we are using the JSON Schema to create the form and the tree representation of the credential Links and this has the side effect that the required field "id" (which will hold the DID) can't be yet provided at this stage of the flow, since it is yet unknown to whom the credential will be directed. Since it's required schema-wise, we have to artificially remove it to prevent validation errors. As you can see [here](https://github.com/0xPolygonID/sh-id-platform/blob/develop/ui/src/components/credentials/IssueCredentialForm.tsx#L36-L42) it's also removed from the form when creating the links.